python/lsst/ip/isr/photodiode.py

```diff
@@ -169,15 +169,38 @@ def fromTable(cls, tableList):
             The calibration defined in the tables.
         """
         dataTable = tableList[0]
-
         metadata = dataTable.meta
+
+        # Dump useless entries that are carried over from merging
+        # HDU[0]'s header with the header from HDU[1] (which has the
+        # data table).
```

Comment:
Is this still necessary with the `merge_headers` tooling?

Reply:
Yes, but Astropy provides an API for this so I think when the header is first read in we need to call `primaryHeader.strip()` on it and then these lines won't be needed.

Reply:
Is this comment still relevant?